nucypher/config/storages.py

```diff
@@ -189,8 +189,9 @@ def __init__(self, parent_dir: str = None, *args, **kwargs) -> None:
         self.__temporary_certificates = list()
         self._temp_certificates_dir = tempfile.mkdtemp(prefix='nucypher-temp-certs-', dir=parent_dir)
 
-    def __del__(self):
-        shutil.rmtree(self._temp_certificates_dir, ignore_errors=True)
+    # Pending fix for 1554.
```

Comment:
```suggestion
    # TODO: Pending fix for 1554.
```
Helps find this line with IDEs when looking for TODOs.

Reply:
Yeah, I initially had it as a TODO, but it's actually not a TODO.  I mean, isn't a TODO something that can be acted presently?   This is something we're waiting on a fix for.  

I think I have generally refrained from putting `TODO` near upstream defects, but that might not be a good approach.

Reply:
Oh no, nevermind.  My bad.